src/o365/spo/commands/site/site-remove.ts

```diff
@@ -46,6 +51,10 @@ class SpoSiteClassicRemoveCommand extends SpoCommand {
   }
 
   public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    if (cmd.commandWrapper.command.indexOf(SITE_CLASSIC_REMOVE) === 0) {
```

Comment:
The `showDeprecationWarning` method already contains the logic to detect if the command was called using the deprecated name so it's not needed to duplicate this logic here.